portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,77 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* 
+ * If user is not logged in, create a login button that will allow the user to
+ * login and add it to the DOM. Otherwise, create the comment input form and 
+ * the button to restrict number of comments and add them to the DOM.
+ */
+function getLogin() {
+    fetch("/login")
+    .then(response => response.json())
+    .then((result) => {
+        const button_area = document.getElementById("button-area");
+        const msg_form = document.getElementById("msg-form");
+        button_area.innerHTML = '';
+        let button = document.createElement('button');
```

Comment:
Some strings literals are using double quote while other are using single quotes. It's good to be consistent. The Google JavaScript Style is recommending single quotes.

Reference: https://google.github.io/styleguide/jsguide.html#features-string-literals